src/app/components/Application/Main.jsx

```diff
@@ -39,7 +51,13 @@ class Main extends React.Component {
    */
   getNewPickSet(picks, selectedFilters) {
     if (!selectedFilters || !selectedFilters.length) {
-      return picks || [];
+      const updatedPicks =
+        this.filterByAudience(picks, this.props.currentAudience, 'staff-picks');
```

Comment:
Whenever we get a new pick set, we always want to filter it by audience type. The `'staff-picks'` should be updated later.